milabench/metrics/archive.py

```diff
@@ -0,0 +1,17 @@
+from milabench.utils import multilogger
+from milabench.testing import replay_run
+
+
+def publish_archived_run(backend, folder):
+    """Publish an archived run to a database"""
+
+    # TODO:
+    #   - enrich old runs with some missing info ?
+    #   - do we validate the run before pushing it ?
```

Comment:
Should we make sure we have metadata populated or check if we have rates

Reply:
Metadata, yes, because the data is useless if we have no information about the system on which the benchmarks ran. If there are no rates, then there will be no rows for that key and we can raise the issue in the report.

Reply:
Closing the conversation as there is nothing additional to do :)